tests/Avalonia.Markup.Xaml.UnitTests/Xaml/ControlTemplateTests.cs

```diff
@@ -308,6 +309,16 @@ public void ControlTemplate_With_Panel_Children_Are_Added()
             Assert.Equal("Foo", foo.Name);
             Assert.Equal("Bar", bar.Name);
         }
+
+        [Fact]
+        public void ControlTemplate_Can_Be_Empty()
+        {
+            var xaml = "<ControlTemplate xmlns='https://github.com/avaloniaui' />";
+            var template = AvaloniaRuntimeXamlLoader.Parse<ControlTemplate>(xaml);
+
+            var templateResult = template.Build(new TemplatedControl());
+            Assert.Null(templateResult);
```

Comment:
Hmm, not actually sure that this _should_ be legal but yeah seems to the the current behavior.